setup.md

```diff
@@ -174,44 +174,206 @@ before pushing them to GitHub.
 
 In order to preview changes locally, you must install the software described below.
 
-1.  **[Ruby](https://www.ruby-lang.org/en/downloads/)**.
-
-    **Linux/macOS**: Ruby is usually included with Linux and macOS ---
-    this is usually sufficient to preview your changes locally.
-
-    If you aren't able to preview reliably with your default Ruby installation,
-    you can use the same version of Ruby as GitHub. Currently,
-    GitHub uses Ruby 2.5.3. In order to install Ruby 2.5.3 on Linux and macOS, we recommend using
-    [rbenv](https://github.com/rbenv/rbenv):
-
+### Windows
+
```

Comment:
Here all options that exist in Windows have to be introduced and then explained below.